### ISSUE TITLE
coreutils: date doesn't respect TZ 

### ISSUE THREAD

**Issue by xu-cheng** [Homebrew/linuxbrew#390]
```
$ /bin/date # system date command
Tue May  5 21:12:10 HKT 2015
$ ~/usr/bin/date # linuxbrew date command
Tue May  5 13:12:16 UTC 2015 # I don't know why but it's UTC
$ TZ="Aisa/Hong_Kong" ~/usr/bin/date
Tue May  5 13:12:30 Aisa 2015 # time is wrong (the value is still in UTC).
```

**Comment by sjackman** [comment]
The TZ environment variable has a funny format. See http://www.gnu.org/software/libc/manual/html_node/TZ-Variable.html
It's looking for the file
```sh
`brew --prefix glibc`/etc/localtime
```
which doesn't exist. Try instead `export TZ=:/etc/localtime`

```sh
$ date
Wed May  6 00:32:21 UTC 2015
$ TZ=PST8PDT date
Tue May  5 17:32:26 PDT 2015
$ TZ=:/etc/localtime date
Tue May  5 17:32:38 PDT 2015
```

**Comment by sjackman** [comment]
The `glibc` postinstall should probably create a `$(brew --prefix glibc)/etc/localtime` file.

**Comment by sjackman** [comment]
Even easier workaround:
```sh
ln -s /etc/localtime `brew --prefix glibc`/etc/
```

**Comment by sjackman** [comment]
Fixed by c569aa8d12cb6566b70968fb5f82b1c272edecf6

**Comment by sjackman** [comment]
Thanks for the bug report, Xu.
